_data/glossary/customresourcedefinition.yaml

```diff
@@ -0,0 +1,10 @@
+id: Custom Resource Definition
+name: Custom Resource Definition
+tags:
+- fundamental
+- operation
+full-link: docs/tasks/access-kubernetes-api/migrate-third-party-resource/
+short-description: >
+  A custom resource definition is the representation of a customization of a particular Kubernetes installation.
+long-description: >
+  A resource definition that defines the endpoint in the Kubernetes API that stores a collection of API objects of a certain kind that is specified.
```

Comment:
Something like: `Creating a CustomResourceDefinition creates a new RESTful API endpoint which can be used to create and manage custom resources.`?

We should also have a glossary term for custom resources (and we could link it here). Most users who search for CRDs also search for custom resources.